microhapulator/data/template.html

```diff
@@ -97,13 +97,9 @@ <h1>MicroHapulator Report</h1>
 
             <h2>Table of Contents</h2>
             <ol>
-                <li><a href="#readqc">Read QA/QC</a></li>
-                {% if reads_are_paired %}
-                <li><a href="#readmerging">Read Merging</a></li>
-                {% endif %}
-                <li><a href="#readmapping">Read Mapping</a></li>
-                <li><a href="#typing">Haplotype Calling</a></li>
-                <li><a href="#filters">Genotype Calling</a></li>
+                {% block table_of_contents %}
+                {% endblock %}
+
```

Comment:
To eliminate the if statements handling the differences between the table of contents for paired versus single ended reads, `{% block table_of_contents %} {% endblock %} `was added to the main `template.html`. These same tags were added to `paired.html` and `single.html` with the specific templates for each in between the tags. This same pattern is used throughout to handle every case where paired end and single end reads have different contents in the report.